cmd/liqoctl/cmd/peer.go

```diff
@@ -42,6 +44,14 @@ cross-cluster pod-to-pod communication. With the *in-band* approach, on the othe
 hand, all control plane traffic flows inside the VPN tunnel. The latter approach
 relaxes the connectivity requirements, although it requires access to both
 clusters (i.e., kubeconfigs) to start the peering process and setup the VPN tunnel.
+
+This command enables a peering to an already known remote cluster without the
+needing of specifying the approach. It will use the same approach already used.
```

Comment:
```suggestion
This command enables a peering towards an already known remote cluster, without the
need of specifying all authentication parameters. It adopts the same approach already used
while peering for the first time with the given remote cluster.
```

Just a proposal though.